Neos.ContentRepository.Core/Classes/NodeType/NodeType.php

```diff
@@ -508,179 +485,7 @@ public function getTypeOfAutoCreatedChildNode(NodeName $nodeName): ?NodeType
     public function allowsChildNodeType(NodeType $nodeType): bool
```

Comment:
hmm im wondering since the `allowsGrandchildNodeType` method is gone if this method should stay here in this place?

Its also not often used in userland.